MAINTAINERS.md

```diff
@@ -0,0 +1,66 @@
+
+Maintainers will check in with each other before making changes to the milestones. Maintainers will focus development and review efforts towards issues and PRs added in milestones, with the closest milestone getting the highest priority.
+
+### Community PRs
+
+For PRs that come in from the community the maintainers will discuss and determine if the PR is in line whith the project's [guiding principlies](#guiding-principles). If the PR is accepted to move forward the maintainers will add it to the appropriate mileston.
+
+### Bug fixes
+
+Bugs can be discovered at any time so its important to have flexibility to prioritize bug fixes as needed. Maintainers will discuss bug fixes received from the community, and if needed, can add them to a milestone to ensure they get reviewed and merged sooner. If maintainers discover bugs during the course of their work and the fix is simple, the maintainer is encouraged to submit a PR for it and discuss prioritizing it with the other maintainers. If a bug affects the security of the projects, maintainers must first discuss privately before fixing and disclosing.
+
+## Changes in maintainership
+
+### Appointing new maintainers
+
+SPIRE Helm Charts maintainers are appointed according to the [process described in the governance document][2].
+
+### Stepping down policy
+
+Maintainers may step down at any time. After you've informed other maintainers, create a pull request to remove yourself from the [CODEOWNERS](CODEOWNERS) file.
+
+### Removal of inactive maintainers
+
+Existing maintainers can be removed from the list if they do not show significant activity on the project. Periodically, the maintainers review the list of maintainers and their activity over the last six months.
+
+If a maintainer has shown insufficient activity over this period, the project manager will contact the maintainer to ask if they want to continue being a maintainer. If the maintainer decides to step down as a maintainer, they open a pull request to be removed from the [CODEOWNERS](CODEOWNERS) file.
+
+If the maintainer wants to remain a maintainer, but is unable to perform the required duties they can be removed with a vote by the [SSC][3] and at least 66% of the current maintainers. A slack messages is sent to the maintainer channel, inviting maintainers of the project to vote. The voting period is five business days. Issues related to a maintainer's performance should be discussed with them among the other maintainers so that they aren't surprised by a pull request removing them.
```

Comment:
```suggestion
If the maintainer wants to remain a maintainer, but is unable to perform the required duties they can be removed with a vote by the [SSC][3] and at least 66% of the current maintainers. A slack message is sent to the maintainer channel, inviting maintainers of the project to vote. The voting period is five business days. Issues related to a maintainer's performance should be discussed with them among the other maintainers so that they aren't surprised by a pull request removing them.
```